.github/workflows/prepare_release.yml

```diff
@@ -56,10 +56,16 @@ jobs:
           import json
           with open("src/main/resources/version-config.json") as version_config:
             schemas = json.load(version_config)
-            version = [schema["kotlinVersion"] for schema in schemas if schema["providerName"] == "${{ matrix.provider }}"][0]
+            versions = [
+              schema["kotlinVersion"]
+              for schema
+              in schemas
+              if schema["providerName"] + schema["kotlinVersion"].split(".")[0]  == "${{ matrix.provider }}"
+            ]
```

Comment:
(minor)

List comprehension formatted this way feels unnatural. Could you format it in IntelliJ and paste here?